src/xrCore/xrCore.h

```diff
@@ -102,7 +102,7 @@
 #endif
 
 #include <time.h>
-#define MODULE_NAME "xrCore.dll"
+#define MODULE_NAME "xrCore"
```

Comment:
I already read about LoadModule(), as well as GetModuleHandle() and some other functions, but I dont know all MODULE_NAME usages. May be it's not just those methods argument. I have no IDE, nor the source code ATM on this pc.